File: Python/tests/test_bart_5bias.py
import pytest

from hbayesdm.models import bart_5bias


def test_bart_5bias():
    _ = bart_5bias(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_bart_5bias.R
context("Test bart_5bias")
library(hBayesDM)

test_that("Test bart_5bias", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_5bias(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
